trlx/models/modeling_ppo.py

```diff
@@ -303,23 +315,30 @@ def forward(
     def generate(self, *args, **kwargs) -> Union[ModelOutput, torch.LongTensor]:
         return self.base_model.generate(*args, **kwargs)
 
-    def state_dict(self, *args, **kwargs):
+    def state_dict(self, heads_only=False, *args, **kwargs):
         """
         Returns the state dictionary of the model. We add the state dictionary of the value head
         to the state dictionary of the wrapped model by prepending the key with `v_head.`.
         """
-        base_model_state_dict = self.base_model.state_dict(*args, **kwargs)
         v_head_state_dict = self.v_head.state_dict(*args, **kwargs)
```

Comment:
I haven't tried to run it with accelerate, and I have trouble reproducing it in Google Colab's because of other environment problems, but from your error log the problem comes from having `heads_only` before `*args`. I just pushed a correction. Do you still get an error ? Don't hesitate to double-check the functions save and load in accelerate_base_trainer.py.